### ISSUE TITLE
Problem CMI and ItemsAdder getting ☐ instead of chat images.

### ISSUE THREAD

**Issue by N1D4L0S** [Zrips/CMI#8049]
### Description of Issue

Hey,
Ive already looked in the Itemsadder github and pretty much everywhere about this problem..
Ive disabled the ClickHoverMessage variable in CMI chat and updated all of my plugins and paper but the bug still persist.
Ive tried without optifine, shaders, clientside resourcespack.
Ive already looked into client log and found nothing.

![2023-08-04_15 29 27](https://github.com/Zrips/CMI/assets/45001265/ed195796-f00a-4a63-bd54-cfdb40e9f777)
![2023-08-04_15 28 58](https://github.com/Zrips/CMI/assets/45001265/247de0c3-b44b-4f83-a577-43977ee37e80)

### Version Information

```shell
9.6.2.3
```

### Errors

```shell
none
```

### Agreements

- [X] My server is supported by CMI.
- [X] My version of CMI at the time of this report is up to date.
- [X] I have searched the github and asked around before making this report.

**Comment by CloudeLecaw** [comment]
as far as i know cmi does not suoport these images

**Comment by kainlighty** [comment]
Yes, it does not accept images. Try using placeholders. And the author of Oraxen recommended using papi -> utils expansion, try it, maybe it will work for you on itemsadder too
